modules/ROOT/pages/configure-host.adoc

```diff
@@ -4,22 +4,33 @@ include::_attributes.adoc[]
 endif::[]
 
 Your organization is considered the host partner in Anypoint Partner Manager.
+Within the Host page, which is named after your organization, you can do the following:
 
-Within host page you can:
+* Add B2B identifiers to uniquely identify yourself with your partners.
```

Comment:
Remove the first word (verb) from each bullet list item, to grammatically comply with the suggested intro sentence.

Reply:
Fixed some of this. Will do the rest on the last pass.